Tests/MeiliSearchIntegrationTests/UpdatesTests.swift

```diff
@@ -105,11 +105,8 @@ class UpdatesTests: XCTestCase {
         self.client.getAllUpdates(UID: self.uid) { result in
 
             switch result {
-            case .success(let updates):
-                let statuses: [String] = ["enqueued", "processed", "fail"]
-                updates.forEach { (update: Update.Result) in
-                    XCTAssertTrue(statuses.contains(update.status))
-                }
+            case .success:
+                break
```

Comment:
Sorry to come back to this again! But can't we check at least the type of `let updates`? So that we check the returned value of the method. If someone change it in the future we would be able to see it.